libs/common/src/typegraph/types.rs

```diff
@@ -92,11 +92,11 @@ pub struct NumberTypeData {
 #[derive(Serialize, Deserialize, Clone, Debug)]
 #[serde(rename_all = "camelCase")]
 pub struct IntegerTypeData {
-    pub minimum: Option<i64>,
-    pub maximum: Option<i64>,
-    pub exclusive_minimum: Option<i64>,
-    pub exclusive_maximum: Option<i64>,
-    pub multiple_of: Option<i64>,
+    pub minimum: Option<i32>,
```

Comment:
I recall i64 is limited by wasm, add maybe a comment to explain the i32 choice here?